Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -141,7 +111,45 @@ export async function run(artifactToolPath: string): Promise<void> {
             packageVersion: version,
         } as artifactToolRunner.IArtifactToolOptions;
 
-        publishPackageUsingArtifactTool(publishDir, publishOptions, toolRunnerOptions);
+        publishStatus = publishPackageUsingArtifactTool(publishDir, publishOptions, toolRunnerOptions);
+
+        //If package already exist, retry with a newer version
+        if (publishStatus != null && publishStatus == 17) {
+            let newVersion: string;
+
+            if (feedType == "internal") {
+                [serviceUri, packageName, feedId, projectId, accessToken] = authSetup(feedType);
+
+                internalAuthInfo = new auth.InternalAuthInfo([], accessToken);
+                toolRunnerOptions.env.UNIVERSAL_PUBLISH_PAT = internalAuthInfo.accessToken;
+            }
+
+            else {
+                [serviceUri, packageName, feedId, projectId, accessToken] = authSetup(feedType);
+
+                //Catch the no external point error
+                if (!serviceUri) {
+                    return
```

Comment:
Should we log something to indicate no external point error?

Reply:
It is log here before function returns above.

https://github.com/microsoft/azure-pipelines-tasks/blob/d54f17ebe0c138980e61dffbabe7ec62d1eec5d5/Tasks/UniversalPackagesV0/universalpublish.ts#L271